tests/conftest.py

```diff
@@ -36,10 +34,43 @@ def start_storage():
     )
 
 
+def _wait_for_es():
+    from elasticsearch import Elasticsearch
+
+    es = Elasticsearch(hosts='http://localhost:9200/')
+    while not es.ping():
+        time.sleep(0.5)
+
+
+def _wait_for_milvus():
+    from pymilvus import connections, has_collection
+    from pymilvus.exceptions import MilvusUnavailableException
+
+    milvus_conn_alias = f'pytest_localhost_19530'
+    connections.connect(alias=milvus_conn_alias, host='localhost', port=19530)
+    milvus_ready = False
+    while not milvus_ready:
+        try:
+            has_collection('ping', using=milvus_conn_alias)
+            milvus_ready = True
+        except MilvusUnavailableException as e:
+            time.sleep(0.5)
+
+
 @pytest.fixture(scope='session')
 def set_env_vars(request):
     _old_environ = dict(os.environ)
     os.environ.update(request.param)
     yield
     os.environ.clear()
     os.environ.update(_old_environ)
+
+
+@pytest.fixture
+def milvus_cleanup():
+    yield
+    from pymilvus import list_collections, drop_collection
+
+    alias = f'docarray_localhost_19530'  # assumes default host and port are used
+    for c in list_collections(using=alias):
+        drop_collection(c, using=alias)
```

Comment:
I wouldn't care about the clean up, the tester should just remove the docker container + collections between tests should have different names (unless, keeping all collections in the same testing session hugely impacts the test performance)

Reply:
You are right, this is not used anymore anyways. will remove